conjur/cli.py

```diff
@@ -129,7 +129,7 @@ def run(self, *args):
         global_optional.add_argument('-h', '--help', action='help', help="Display help list")
         global_optional.add_argument('-v', '--version', action='version',
                                      help="Display version number",
-                                     version='Conjur Python CLI version ' + __version__ + "\n"
+                                     version='Conjur CLI version ' + __version__ + "\n"
```

Comment:
please remove the © from the copyright function

Reply:
This should be fine as-is as long as the file has the `# -*- coding: utf-8 -*-` at the top

Reply:
The copyright was found on the top of `cli.py` which has the `# -*- coding: utf-8 -*-`. When we packed the executable on a RHEL/Windows machine and ran it on a RHEL/Windows machine we were getting the following error:

<img width="1003" alt="Screen Shot 2020-12-07 at 17 47 29" src="https://user-images.githubusercontent.com/19418506/102178473-20c5d600-3eae-11eb-8724-1bb8573cb503.png">

xa9 is the [copyright](https://www.codetable.net/hex/a9) so I thought best to remove it since we say Copyright anyways